site/styles/_footer.scss

```diff
@@ -80,6 +86,29 @@ footer {
   }
 }
 
+.social-link-button {
+  align-items: center;
+  background-color: var(--footer-social-bg);
+  border-radius: 50%;
+
+  display: flex;
+  height: 32px;
+  justify-content: center;
+  width: 32px;
+
+  &:hover {
+    background-color: var(--footer-social-bg-hover);
+
+    svg path {
```

Comment:
This is how spaghetti is made!  If somebody in the future were to search for `svg path`, they would see it in two places instead of one!

A better approach would be something like:
```
.social-link-button {
  ... stuff...

  &:hover {
    background-color: var(--footer-social-bg-hover);
  }

  svg path {
    fill: var(--footer-social-icon);

    &:hover {
      fill: var(--footer-social-icon-hover);
    }
  }
}
```

It's easier to maintain despite more lines, and it's waaaay easier to deal with when you are writing overrides in a custom shift72-template.

Reply:
Ah yep that makes sense

Reply:
Fixed this by setting the colour for the buttons themselves instead of the SVG path